src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +19,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(s -> Arrays.stream(s.split(",")))
+                .filter(n -> Integer.parseInt(n) % 2 == 0)
```

Comment:
```suggestion
                .filter(number -> Integer.parseInt(n) % 2 == 0)
```

I would recommend not using an abbreviation if you can give the full name

Reply:
ok ok